.github/workflows/docs.yml

```
@@ -36,26 +35,39 @@ jobs:
         id: expo-caches
         with:
           yarn-docs: 'true'
+          yarn-tools: 'true'
+          yarn-workspace: 'true'
       - name: ➕ Add `bin` to GITHUB_PATH
         run: echo "$(pwd)/bin" >> $GITHUB_PATH
+      - name: 📝 Regenerate `unversioned` data for Docs
+        run: et gdad
```

Shortcuts are nice, but in code it seems more clear if we only use full command names
```suggestion
        run: expotools generate-docs-api-data
```